src/value/borrowed.rs

```diff
@@ -22,28 +20,24 @@ pub type Object<'v> = HashMap<Cow<'v, str>, Value<'v>>;
 /// As we reference parts of the input slice the resulting dom
 /// has the dame lifetime as the slice it was created from.
 pub fn to_value<'v>(s: &'v mut [u8]) -> Result<Value<'v>> {
-    let de = stry!(Deserializer::from_slice(s));
-    BorrowDeserializer::from_deserializer(de).parse()
+    match Deserializer::from_slice(s) {
+        Ok(de) => Ok(BorrowDeserializer::from_deserializer(de).parse()),
+        Err(e) => Err(e),
+    }
 }
 
 /// Borrowed JSON-DOM Value, consider using the `ValueTrait`
-/// to access it'scontent
+/// to access it's content
```

Comment:
the apostrophe in the comment can go, `its` vs `it's` ... yay English!

Reply:
fixed!